pkg/planner/core/planbuilder.go

```diff
@@ -3347,15 +3353,16 @@ func (b *PlanBuilder) buildSimple(ctx context.Context, node ast.StmtNode) (Plan,
 	case *ast.KillStmt:
 		// All users can kill their own connections regardless.
 		// If you have the SUPER privilege, you can kill all threads and statements unless SEM is enabled.
-		// In which case you require RESTRICTED_CONNECTION_ADMIN to kill connections that belong to RESTRICTED_USER_ADMIN users.
+		// In which case you require RESTRICTED_USER_ADMIN to kill connections that belong to RESTRICTED_USER_ADMIN users.
```

Comment:
Yes, the RESTRICTED_CONNECTION_ADMIN permission will be deprecated, and it will be replaced by the RESTRICTED_USER_ADMIN permission for unified management.